Documentation/ApiOverview/Events/Events/Frontend/_AfterContentHasBeenFetchedEvent/_MyEventListener.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyVendor\MyExtension\Frontend\EventListener;
+
+use TYPO3\CMS\Core\Attribute\AsEventListener;
+use TYPO3\CMS\Frontend\Event\AfterContentHasBeenFetchedEvent;
+
+final class MyEventListener
```

Comment:
In one of the other patches you have used "readonly"

```suggestion
final readonly class MyEventListener
```